index.bs

```diff
@@ -73,47 +73,42 @@ spec: webidl
   </h2>
   <p><em>This section is non-normative.</em></p>
   <p>
-    This document's goal is to specify an API that will help developers to
-    handle permissions on the Web platform. Web APIs have different ways to
-    deal with permissions. The [[notifications]] API allows developers to
+    This document specifies an API to query and request permissions to
+    powerful features on the Web platform. Web APIs have different ways to
+    deal with permissions. The [[Notifications]] API allows developers to
     request a permission and check the permission status explicitly. Others
-    might only expose the status to web pages when they try to use the API,
-    like the [[geolocation-API]] which fails if the permission was not
+    expose the status to web pages when they try to use the API,
+    like the [[Geolocation-API]] which fails if the permission was not
```

Comment:
Both of them match; it's Bikeshed saying that we should use a consistent spelling of a `[[reference]]` within a given spec.